packages/tailwind/src/utils/config-handler.ts

```diff
@@ -40,13 +40,19 @@ const toShortCutStyles = (theme: ThemeOptions, name: string, color: string) =>
         },
         ...AVAILABLE_VARIANTS.map((variant) => ({
           name: () => `${name}-${size}${CLASS_NAME_SEPARATOR}${variant}`,
-          styles: () => `${VARIANT_CLASSES[variant]}-${color}`
+          styles: () => `${prefix}${VARIANT_CLASSES[variant]}-${color}`
         }))
       ])
     )
   );
 
-const generateConfig = <T extends ThemeOptions>(theme: T, parsedColors: ParsedColors) => {
+const generateConfig = <T extends ThemeOptions>(
+  theme: T,
+  parsedColors: ParsedColors,
+  prefix?: string
+) => {
+  const prefixToAdd = prefix ? prefix : "";
```

Comment:
We can move the initialization of the prefix to line 52 change it to `prefix: string = ''` and use the prefix across the code, avoiding to create this new variable